nautobot/docs/user-guide/administration/installation/install_system.md

```diff
@@ -40,406 +40,444 @@ You must select either MySQL (MariaDB is **not** supported) or PostgreSQL. Postg
-    You are now connected to database "nautobot" as user "postgres".
-    nautobot=# GRANT CREATE ON SCHEMA public TO nautobot;
-    GRANT
-    nautobot=# \q
-    ```
+        As root, edit `/var/lib/pgsql/data/pg_hba.conf` and change `ident` to `md5` for the lines below.
 
-    ### Verify PostgreSQL Service Status
+        Before:
 
-    You can verify that authentication works issuing the following command and providing the configured password. (Replace `localhost` with your database server if using a remote database.)
+        ```no-highlight
+        # IPv4 local connections:
+        host    all             all             127.0.0.1/32            ident
+        # IPv6 local connections:
+        host    all             all             ::1/128                 ident
+        ```
 
-    If successful, you will enter a `nautobot` prompt. Type `\conninfo` to confirm your connection, or type `\q` to exit.
+        After:
 
-    ```no-highlight
-    psql --username nautobot --password --host localhost nautobot
-    ```
+        ```no-highlight
+        # IPv4 local connections:
+        host    all             all             127.0.0.1/32            md5
+        # IPv6 local connections:
+        host    all             all             ::1/128                 md5
+        ```
 
-    Example output:
+        #### Start PostgreSQL
 
-    ```no-highlight
-    Password for user nautobot:
-    psql (10.15)
-    Type "help" for help.
+        Start the service and enable it to run at system startup:
 
-    nautobot=> \conninfo
-    You are connected to database "nautobot" as user "nautobot" on host "localhost" (address "127.0.0.1") at port "5432".
-    nautobot=> \q
-    ```
+        ```no-highlight
+        sudo systemctl enable --now postgresql
+        ```
 
-=== "RHEL8 MySQL"
+        #### Create a PostgreSQL Database
 
-    ### Install MySQL
+        At a minimum, we need to create a database for Nautobot and assign it a username and password for authentication. This
+        is done with the following commands.
 
-    This will install the MySQL database server and client. Additionally, MySQL requires that `gcc` and the MySQL development libraries are installed so that we may compile the Python `mysqlclient` library during the Nautobot installation steps.
+        !!! danger
+            **Do not use the password from the example.** Choose a strong, random password to ensure secure database authentication for your Nautobot installation.
 
-    ```no-highlight
-    sudo dnf install -y gcc mysql-server mysql-devel
-    ```
+        ```no-highlight
+        sudo -u postgres psql
+        ```
 
-    ### Start MySQL
+        Create the database and grant permissions to the Nautobot user.
 
-    Start the service and enable it to run at system startup:
+        ```no-highlight
+        CREATE DATABASE nautobot;
+        CREATE USER nautobot WITH PASSWORD 'insecure_password';
+        GRANT ALL PRIVILEGES ON DATABASE nautobot TO nautobot;
+        \q
```

Comment:
missing?

```suggestion
        \connect nautobot
        GRANT CREATE ON SCHEMA public TO nautobot;
        \q
```

Reply:
Will update. This was what was there previous.